docs/external-node.md

```diff
@@ -265,56 +265,69 @@ please refer to the [getting-started guide](getting-started.md#open-vswitch).
    make docker-bin
    ```
 
-2. The `antrea-agent.conf` file specifies agent configuration parameters. Copy
-   the [agent configuration file](../build/yamls/externalnode/conf/antrea-agent.conf)
-   to the VM and edit the `antrea-agent.conf` file to set `clientConnection`,
-   `antreaClientConnection` and `externalNodeNamespace` with the correct values.
-   Copy `antrea-agent.antrea.kubeconfig` and `antrea-agent.kubeconfig` files to
-   the VM, that were generated in the step 4 and step 5 of
-   [Prerequisites on Kubernetes cluster](#prerequisites-on-kubernetes-cluster).
-
-   ```bash
-   AGENT_NAMESPACE="vm-ns"
-   AGENT_CONF_PATH="/etc/antrea"
-   mkdir -p $AGENT_CONF_PATH
-   # Copy antrea-agent kubeconfig files
-   cp ./antrea-agent.kubeconfig $AGENT_CONF_PATH
-   cp ./antrea-agent.antrea.kubeconfig $AGENT_CONF_PATH
-   # Update clientConnection and antreaClientConnection
-   sed -i "s|kubeconfig: |kubeconfig: $AGENT_CONF_PATH/|g" antrea-agent.conf
-   sed -i "s|#externalNodeNamespace: default|externalNodeNamespace: $AGENT_NAMESPACE|g" antrea-agent.conf
-   # Copy antrea-agent configuration file
-   cp ./antrea-agent.conf $AGENT_CONF_PATH
-   ```
-
-3. Create `antrea-agent` service. Note: environment variable `NODE_NAME` is set
-   in the service configuration, if the VM's hostname is different from the name
-   defined in the `ExternalNode` resource. Below is a sample snippet to start
-   `antrea-agent` as a service on Ubuntu 18.04 or later:
-
-   ```bash
-   AGENT_BIN_PATH="/usr/sbin"
-   AGENT_LOG_PATH="/var/log/antrea"
-   mkdir -p $AGENT_BIN_PATH
-   mkdir -p $AGENT_LOG_PATH
-   cat << EOF > /etc/systemd/system/antrea-agent.service
-   Description="antrea-agent as a systemd service"
-   After=network.target
-   [Service]
-   Environment="NODE_NAME=vm1"
-   ExecStart=$AGENT_BIN_PATH/antrea-agent \
-   --config=$AGENT_CONF_PATH/antrea-agent.conf \
-   --logtostderr=false \
-   --log_file=$AGENT_LOG_PATH/antrea-agent.log
-   Restart=on-failure
-   [Install]
-   WantedBy=multi-user.target
-   EOF
+2. Copy configuration files to the VM, including [antrea-agent.conf](../build/yamls/externalnode/conf/antrea-agent.conf),
+   which specifies agent configuration parameters;
+   `antrea-agent.antrea.kubeconfig` and `antrea-agent.kubeconfig`, which were
+   generated in steps 4 and 5 of [Prerequisites on Kubernetes cluster](#prerequisites-on-kubernetes-cluster).
+
+3. Bootstrap `antrea-agent`
+  * Bootstrap `antrea-agent` using the [installation script](../hack/externalnode/install-vm.sh)
+    as shown below and skip the next step.
```

Comment:
What does "the next step" refer to? Should we just remove "and skip the next step"?

Reply:
The next step referred to bootstrapping manually. As it is not clear, will remove it.
Updating it based on below suggestion.